lib/bundler/cli.rb

```diff
@@ -148,13 +152,13 @@ def check
 
       If the bundle has already been installed, bundler will tell you so and then exit.
     D
-    method_option "binstubs", :type => :string, :lazy_default => "bin", :banner =>
+    deprecated_option "binstubs", :type => :string, :lazy_default => "bin", :banner =>
       "Generate bin stubs for bundled gems to ./bin"
     method_option "clean", :type => :boolean, :banner =>
       "Run bundle clean automatically after install"
```

Comment:
Can we deprecate this as well? I am pretty sure we can say "you can just run `clean`, which will `install` and then `clean`". The concept of cleaning isn't even really coherent without an install happening first, imo.